src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -37,6 +39,24 @@ def list(self):
         fsmap = mgr.get("fs_map")
         return fsmap['filesystems']
 
+    def create(self, name: str, service_spec: Dict[str, Any]):
```

Comment:
Can you add the API tests for this endpoint and  a unittest for cephfs service in frontend as well?